bw2data/revisions.py

```diff
@@ -33,10 +33,14 @@
 from bw2data.snowflake_ids import snowflake_id_generator
 from bw2data.utils import get_node
 
-try:
-    from typing import Self
-except ImportError:
```

Comment:
Changing this because `mypy` complains about the `ImportError`:

```
$ env-3.9/bin/mypy --follow-imports skip --disable-error-code import-untyped bw2data/revisions.py
[…]
bw2data/revisions.py:37: error: Module "typing" has no attribute "Self"  [attr-defined]
bw2data/revisions.py:37: note: Use `from typing_extensions import Self` instead
bw2data/revisions.py:37: note: See https://mypy.readthedocs.io/en/stable/runtime_troubles.html#using-new-additions-to-the-typing-module
[…]
```